src/EffectNode.cpp

```diff
@@ -15,105 +15,91 @@
 #include <algorithm>
 #include "Paths.h"
 
-EffectNode::EffectNode(Context *c, QString file)
-    : VideoNode(c)
-    , m_intensity(0)
-    , m_openGLWorker(new EffectNodeOpenGLWorker(this), &QObject::deleteLater)
-    , m_ready(false) {
+EffectNode::EffectNode(Context *context, QString file)
+    : VideoNode(new EffectNodePrivate(context))
+{
+    qRegisterMetaType<Chain>("Chain");
```

Comment:
Why is this here?

Reply:
this is here so it can be passed via `invokeMethod` as an argument (Q_ARG)